src/scss/components/radio/_index.scss

```diff
@@ -0,0 +1,264 @@
+@import  "vars";
+@import  "mixins";
+@import  "components/form/index";
+
+// BASE STYLES
+hx-radio {
+  border-radius: 1rem;
+  display: inline-block;
+  height: 1rem;
+  vertical-align: baseline;
+  width: 1rem;
+}
+
+hx-radio-set {
+  display: block;
+
+  > * {
+    margin: 0.25rem 0; // FIXME: solve with spacing system
+
+    &:first-child {
+      margin-top: 0; // FIXME: solve with spacing system
+    }
+
+    &:last-child {
+      margin-bottom: 0; // FIXME: solve with spacing system
+    }
+  }
+}
+
+// LAYOUT GEOMETRY
+hx-radio-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "ctrl lbl"
+    ".    help";
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto;
+  justify-items: start;
+
+  > * {
+    margin: 0;
+  }
+
+  // facade + text
+  > label {
+    // grid child
+    -ms-grid-column: 2 auto 1fr;
+    -ms-grid-row: 1 auto;
+    grid-area: lbl;
+
+    // grid parent
+    display: -ms-grid;
+    display: grid;
+    grid-gap: 0.5rem;
+    grid-template-areas: "facade txt";
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto;
+    padding: 0;
+
+    > hx-radio {
+      -ms-grid-column: 1;
+      -ms-grid-row: 1;
+      display: none;
+      grid-area: facade;
+      margin-top: 0.25rem;
+    }
+  }
+
+  > p {
+    -ms-grid-column: 2;
+    -ms-grid-row: 2;
+    grid-area: help;
+  }
+
+  > input[type="radio"] {
+    -ms-grid-column: 1;
+    -ms-grid-row: 1;
+    border-radius: 1rem;
+    grid-area: ctrl;
+    height: 1rem;
+    margin: 0.25rem 0.5rem 0 0;
+    width: 1rem;
+
+    &::-ms-check {
+      box-sizing: border-box;
+    }
+  }
+}
+
+// PRISTINE/VALID STATE
+hx-radio-control {
+  > label > hx-radio {
+    background-color: $gray-0;
+    border: 1px solid currentColor;
+    color: $gray-500;
+  }
+
+  > p {
+    //@include HelpText();
+    color: $gray-750;
+    font-size: 0.75rem;
```

Comment:
Please use the `hxHelpText` mixin. Don't inline the mixin styles.

```scss
> p {
  @include hxHelpText;
}
```

(See https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/components/form/mixins/_helpText.scss#L3)

Reply:
Changes are done ... please review